src/test/java/core/basesyntax/strategy/impl/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax.strategy.impl;
+
+import static junit.framework.TestCase.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Transaction;
+import core.basesyntax.strategy.OperationHandler;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BalanceOperationHandlerTest {
+    private static OperationHandler operationHandler;
+
+    @BeforeClass
+    public static void beforeClass() {
+        operationHandler = new BalanceOperationHandler();
+    }
+
+    @Test
+    public void calculate_Ok() {
+        operationHandler.calculate(new Transaction("banana", 20));
+        Integer expectedResult = 20;
```

Comment:
generating expected data should be before calling tested method (calculate in this case)